### PR TITLE
Netperf: download from generic/deps/ dir instead of website

### PR DESCRIPTION
**Pull request by quanwenli**
Signed-off-by: Wenli Quan <wquan@redhat.com>

ID: 1520722

### CODE REVIEW

**Comment by quanwenli** [comment]
JOB LOG    : /root/avocado/job-results/job-2017-12-04T21.52-9302fa2/job.log
 (1/1) smp_8.4096m.repeat1.repeat1.Host_RHEL.m7.u5.qcow2.virtio_blk.up.virtio_net.RHEL.7.5.x86_64.io-github-autotest-qemu.netperf.jumbo.host_guest: PASS (2696.72 s)

File: generic/tests/cfg/netperf.cfg [hunk 1]
```diff
@@ -55,8 +55,7 @@
     #Test base env configration
     ver_cmd = rpm -q qemu-kvm
     netperf_version = 2.6.0
-    netperf_download_link = ftp://ftp.netperf.org/netperf/archive/netperf-2.6.0.tar.bz2
-    pkg_md5sum = 9654ffdfd4c4f2c93ce3733cd9ed9236
+    netperf_pkg = performance/netperf-2.6.0.tar.bz2
```
Comment (hereischen) [inline generic/tests/cfg/netperf.cfg]: This is ok.
But I think put `performance/` into line 99, leave it just `netperf-2.6.0.tar.bz2` could be better.

Comment (quanwenli) [reply generic/tests/cfg/netperf.cfg]: After discuss with ＠hereischen, I will not modify here, it's ok.

**Comment by hereischen** [comment]
Acked-by: Haotong Chen hachen@redhat.com

**Pull request merged**